lib/hanami/action/params.rb

```diff
@@ -129,7 +184,7 @@ def error_messages(error_set = errors)
       # @example
       #   params.valid? # => true
       def valid?
-        @result.success?
+        errors.empty?
```

Comment:
WDYT about `@result.success? && errors.empty?`?

Reply:
@davydovanton I've considered it, but given when we initialize `Errors` it's already full of errors if `@result.success?` is `false`. Because if there is a validation error, `@result.success?` is `false` and it has error messages that are filled into `Errors`.

If you look at the following prospect:

```
true && true => true # validations ok, no errors added
true && false => false # validations ok, but errors were manually added
false && true => false # impossible because if validations are failed, then we'll have errors filled
false && false => false # validations failed, these errors where copied to `Errors`. There could be potential errors added manually
```

If you look at the right side of the operation (after `&&`), that value is always the same of the result: if it's `true`, then the result is `true`, if `false`, the result is `false`. For this reason, I think `errors.empty?` is enough. Does it makes sense?

Reply:
great. thanks for explanation! 💓